modules/social_features/social_user/social_user.install

```diff
@@ -206,3 +206,18 @@ function social_user_update_8810() {
     'view blocked user',
   ]);
 }
+
+/**
+ * Migrate navigation settings into the correct format.
+ */
+function social_user_update_8811() {
```

Comment:
I think config updates are better in post_update hooks, since we rely on schemas having stabilised? I'm never quite sure though.

Reply:
yeah think you are right.